src/hammer-vlsi/synthesis/yosys/__init__.py

```diff
@@ -2,20 +2,34 @@
 # -*- coding: utf-8 -*-
 #
 # Yosys synthesis plugin for Hammer (as part of OpenROAD-flow installation).
+# mostly adoped from OpenLANE scripts found here: https://github.com/The-OpenROAD-Project/OpenLane/tree/master/scripts
 #
 # See LICENSE for licence details.
 
-import os
 from textwrap import dedent as dd
-from typing import List, Optional, Dict, Any
 
+# from hammer_utils import deepdict
+from hammer_vlsi.vendor import OpenROADTool, OpenROADSynthesisTool
+
+###########
+from hammer_vlsi import HammerTool, HammerToolStep, HammerToolHookAction, HierarchicalMode, TCLTool
+from hammer_vlsi.constraints import MMMCCorner, MMMCCornerType
+from hammer_utils import VerilogUtils, optional_map
+from hammer_vlsi import HammerSynthesisTool
 from hammer_logging import HammerVLSILogging
-from hammer_utils import deepdict
-from hammer_vlsi import HammerToolStep
-from hammer_vlsi.vendor import OpenROADSynthesisTool
+from hammer_vlsi import MMMCCornerType
+import hammer_tech
 
+from typing import Dict, List, Any, Optional
 
-class YosysSynth(OpenROADSynthesisTool):
```

Comment:
Why is this not an `OpenROADSynthesisTool`? Is that class conflicting/need to be deprecated?

Reply:
OpenROADSynthesisTool implements routines for the old way we ran OpenROAD (via Makefile), so it didn't have any useful routines and caused an error when I tried having YosysSynth inherit from it. We could probably just remove it.